pkg/metrics/metrics.go

```diff
@@ -0,0 +1,42 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	// Number of out-of-band modifications overwritten by HCO
+	overwrittenModifications = prometheus.NewCounterVec(
```

Comment:
Do we really need this variable? Maybe we can embbed its fields in `HcoMetrics`?

Reply:
Changed